os/torcx-hosting-custom-remotes.md

```diff
@@ -0,0 +1,79 @@
+# Hosting custom Torcx remotes
+
+## Remotes Overview
+
+A Torcx [remote][torcx-remotes-design] is a collection of addon images for torcx, served from a remote source, which can be fetched by a host and used by [torcx-generator][torcx-overview].
+A remote can be served via any traditional web server and content integrity and authenticity is digitally verified via OpenPGP signatures.
+
+## Content of a remote
+
+A Torcx remote is a collection of static files, providing Torcx addon images over HTTP(S) to Container Linux nodes.
+
+Main elements of a Torcx remote are:
+
+ * base URL: base URL for the remote. Supported protocols are `http` and `https`.
```

Comment:
Is `http` actually supported?  I hit an unsupported schema error while testing it.

Reply:
According to documentation and design references yes, but the code doesn't allow that. I've opened https://github.com/coreos/torcx/pull/128.